regression/strings/smoke_tests/performance.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+import time
+from subprocess import check_output
+
+git_output=check_output(["git", "show", "HEAD"])
+commit=git_output.split('\n')[0]
+
+commit_id=time.strftime("%Y_%m_%d__%H_%M_%S")+"__"+commit[7:]
+
+process = check_output(["grep", "^Runtime\ decision\ procedure", "-R"])
+
+file_name='performance_'+commit_id+'.out'
+print "writing to file", file_name
+f=open(file_name, 'w')
+
+for x in process.split('\n'):
```

Comment:
Does this differ from just piping grep to that file? Try using
```
with open("/tmp/crap", "w") as f:
  subprocess.check_call(..., stdout=f)
```

Note use of `check_call` not `check_output`, as the latter would try to capture `stdout` to a string.